opentelemetry-sdk/src/logs/logger_provider.rs

```diff
@@ -52,6 +52,7 @@ impl opentelemetry::logs::LoggerProvider for SdkLoggerProvider {
     }
 
     fn logger_with_scope(&self, scope: InstrumentationScope) -> Self::Logger {
+        // TODO: Should this check suppression context and return a no-op logger?
```

Comment:
To the comment: could a logger created here potentially be used outside of a suppressed context, when it is created within it?

I can't imagine a situation - if you create a logger in a suppressed context, _you are us - OTel_ - and there isn't really a reason you should pass that back out to a client of ours. On the other hand, if this ever happened it would be gnarly to debug the missing logs.

Reply:
good point! You are right. I'll remove this comment, as we should not check suppression for creating Logger, but only while emitting Logs.